minitest-reporters.gemspec

```diff
@@ -12,6 +12,7 @@ Gem::Specification.new do |s|
   s.summary     = %q{Create customizable Minitest output formats}
   s.description = %q{Death to haphazard monkey-patching! Extend Minitest through simple hooks.}
 
+  s.required_ruby_version = '>= 1.9.3'
```

Comment:
why 1.9.3?  Why not 1.8.7?  It looks like minitest doesn't restrict ruby version.